<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingSystem</class>
 <widget class="QMainWindow" name="TestingSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestingSystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="pictureView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="digitalImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Цифровой</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadedImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Загруженный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognizedImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Распознанный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognizeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Распознать</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>21</width>
      <height>561</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Число строк:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Число столбцов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowsNumLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>61</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="columnsNumLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Группа №</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Студент №</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тест №</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>161</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="groupNumLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="studentNumLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="testNumLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Всего вопросов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прав. ответов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Оценка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalQuestionsLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="correctAnswersLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="QLabel" name="markLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нераспознано</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openImageMenu"/>
    <addaction name="action"/>
    <addaction name="openResultsMenu"/>
    <addaction name="separator"/>
    <addaction name="saveMenu"/>
    <addaction name="saveAsMenu"/>
    <addaction name="separator"/>
    <addaction name="printMenu"/>
    <addaction name="separator"/>
    <addaction name="exitMenu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Тесты и студенты</string>
    </property>
    <addaction name="addTestMenu"/>
    <addaction name="addGroupMenu"/>
    <addaction name="addStudentMenu"/>
    <addaction name="watchTestsMenu"/>
    <addaction name="watchGroupMenu"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutMenu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openImageMenu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть изображение...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="exitMenu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="saveParamMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Параметры сохранения</string>
   </property>
  </action>
  <action name="openResultsMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть результаты...</string>
   </property>
  </action>
  <action name="saveMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveAsMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="recognitionParamMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Параметры распознавания</string>
   </property>
  </action>
  <action name="addTestMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить тест...</string>
   </property>
  </action>
  <action name="aboutMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="watchTestsMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Посмотреть доступные тесты</string>
   </property>
  </action>
  <action name="addGroupMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить группу...</string>
   </property>
  </action>
  <action name="addStudentMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить студента...</string>
   </property>
  </action>
  <action name="watchGroupMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Посмотреть доступные группы</string>
   </property>
  </action>
  <action name="printMenu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Печать...</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Создать изображение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
